dokka-integration-tests/gradle/build.gradle.kts

```diff
@@ -215,5 +219,28 @@ testing {
                 systemProperty("junit.jupiter.execution.parallel.mode.default", "CONCURRENT")
             }
         }
+
+        // Register specific Gradle test runs for each example so the examples can be tested individually.
+        exampleGradleProjectsDir.toPath()
+            .listDirectoryEntries()
+            .map { it.name }
+            .filter {
+                it.endsWith("-example")
+                // note: avoid checks like `isDirectory()` to prevent Gradle registering the files as CC inputs.
```

Comment:
I think it makes sense to explicitly make these directories the CC fingerprint input. These tasks now have an issue where the task list with CC and without is inconsistent:
```bash
# Cache CC for basic project
./gradlew :dokka-integration-tests:gradle:testExampleBasicGradle
# Delete basic example
mv ../../examples/gradle-v2/basic-gradle-example ../../examples/gradle-v2/basic-gradle-example.bak
# The task now runs with CC
./gradlew :dokka-integration-tests:gradle:testExampleBasicGradle
# Now it fails with "Cannot locate tasks" due to reconfiguration
./gradlew :dokka-integration-tests:gradle:testExampleBasicGradle --no-configuration-cache
```

Reply:
Thanks, I've used a ValueSource, and I think it's working as intended now.